src/health/HealthController.ts

```diff
@@ -0,0 +1,17 @@
+import {Controller, Get} from '@nestjs/common';
+import {HealthCheck, HealthCheckResult, HealthCheckService} from '@nestjs/terminus';
+
+@Controller('health')
+export class HealthController {
+    constructor(
+        private readonly health: HealthCheckService,
+    ) {
+    }
+
+    @Get()
+    @HealthCheck()
+    public check(): Promise<HealthCheckResult> {
+        // TODO: Currently empty but can be used to check health or Redis and RabbitMQ
+        return this.health.check([]);
```

Comment:
This does need to return some sort of 200 with a body so k8s healthcheck won't kill it

Reply:
I already added some health checks for Redis, Rabbit, and the API.